CONTRIBUTING-HTML.md

```diff
@@ -300,20 +296,18 @@ This doesn't tell the reader anything about how they should use the element. The
 
 For the HTML editor there are three CSS classes that can be applied to the editor container element. This allows the editor to by taller or shorter than it’s standard height. The classes are as follows:
 
-- `tabbed-shorter` - ~11 visible lines of code
-- `tabbed-standard` - ~14 visible lines of code
-- `tabbed-taller` - ~23 visible lines of code
+- `tabbed-shorter` - ~14 visible lines of code
+- `tabbed-standard` - ~18 visible lines of code
+- `tabbed-taller` - ~30 visible lines of code
 
 Usage is as follows. When adding the meta information for your example, set the `height` property to one of the classes specified above. For example:
 
 ```json
-"abbr": {
-    "exampleCode":
-        "./live-examples/html-examples/inline-text-semantics/abbr.html",
-    "cssExampleSrc":
-        "./live-examples/html-examples/inline-text-semantics/css/abbr.css",
-    "fileName": "abbr.html",
-    "title": "HTML Demo: <abbr>",
+"date": {
```

Comment:
Same here with Prettier overwriting the line lengths, this might get clobbered in future.